packages/algolia-experiences/src/render.tsx

```diff
@@ -257,17 +254,14 @@ function blockToWidget(child: Block, container: HTMLElement): Widget[] {
                 children = ch.children.map(renderChild);
               }
 
-              const attributes = Object.fromEntries(
```

Comment:
is this actually not allowed? from checking caniuse I see older chrome not having this but the other browsers seem ok. Should we have chrome 74 as minimum instead?

Reply:
Just crunched the numbers and we would lose 1% of users :/